src/kps/gui/windows/form/dialogs/MailDialog.java

```diff
@@ -30,10 +37,21 @@
         Priority
     }
 
+    /**
+     * Constructor if you don't have an old mail delivery for your dialog.
+     * @param owner the frames owner
+     * @param simulation the simulation to work from
+     */
     public MailDialog(Frame owner, Simulation simulation) {
```

Comment:
A little unrelated, but the parameters in this class could have some @NonNull and @Nullable.